downstream/assemblies/hub/assembly-validated-content.adoc

```diff
@@ -0,0 +1,75 @@
+
+When you download and run the bundle installer, certified and validated collections are automatically uploaded. 
+Certified colections are uploaded into the `rh-certified` repository, while validated collections are uploaded into the `validated` repository. 
+
+This behavior can be configured with two variables:
+
+[cols="20%,50%",options="header"]
+|====
+| Name | Description 
+| `automationhub_seed_collections` | A boolean that defines whether or not preloading is enabled.
+| `automationhub_collection_seed_repository` | If `automationhub_seed_collections` is set to `true`, this variable enables you to specify the type of content to upload. 
+Possible values are `certified` or `validated`. 
+If missing both content sets will be uploaded.
+|====
+
+== {Valid}
+
+[NOTE]
+====
+* {Valid} is only available with a valid subscription to {PlatformName}.
+* Unlike {CertifiedCon}, {Valid} is not supported by Red Hat or our partners. 
+* From the {PlatformName} {PlatformVers} release, {Valid} is preloaded into {PrivateHubName} and can be updated manually by downloading the packages.
+====
+
+[cols="20%,30%,30%,20%",options="header"]
+|====
+| Entity | Collection name | Description | Published by
+| Ansible | `cloud.azure_roles.load_balancer` | A role to manage Azure Load Balancer | Red Hat Ansible
+| Ansible | `cloud.azure_roles.managed_postgresql` | A role to manage Azure PostGreSQL Database |Red Hat Ansible
+| Ansible | `cloud.azure_roles.network_interface` | A role to manage Azure Network Interface | Red Hat Ansible 
+| Ansible | `cloud.azure_roles.networking_stack` | A role to manage Azure Networking Stack | Red Hat Ansible
+| Ansible | `cloud.azure_roles.resource_group`  | A role to manage Azure Resource Group | Red Hat Ansible
+| Ansible | `cloud.azure_roles.security_group`  | A role to manage Azure Security Group | Red Hat Ansible
+| Ansible | `cloud.azure_roles.virtual_machine`  | A role to manage Azure Virtual Machine | Red Hat Ansible
+| Ansible | `network.base` | A validated content collection to consist base config related implementation that would be utilized by other validated content | Red Hat Ansible
+| Ansible | `network.base` | A validated content collection to configure bgp and provide capabilities to do operational state/healthchecks | Red Hat Ansible
+| Ansible | `network.acls` | A validated content collection to configure acls and provide capabilities to do operational state/healthchecks | Red Hat Ansible
+| Ansible | `network.interfaces` | A validated content collection to configure interfaces and provide capabilities to do operational state/healthchecks | Red Hat Ansible
+| Ansible | `network.ospf` | A validated content collection to configure ospf and provide capabilities to do operational state/healthchecks | Red Hat Ansible
+| Ansible | <name yet to decide> | Connectivity between on-prem network device (for ex CSR) and cloud gateway(for ex AWS) | Red Hat Ansible
```

Comment:
Is this note "<name yet to decide>" applicable to the empty cells as well?

Reply:
I don't know.